### ISSUE TITLE
Resolution of Input Data

### ISSUE THREAD

**Issue by alanacg** [mmann1123/ts-raster#3]
I have been having difficulty applying the calculatefeatures function to my raster files with a resolution of under 30 m (they are also between 2-4 GB). I was wondering if this is related to the code library, or my device. Is there a resolution limit for the rasters that can be analyzed with this library? Thanks!

**Comment by mmann1123** [comment]
Yeah this implementation was really memory intensive. I have been working
on an xarray implementation.

**Comment by alanacg** [comment]
Good to hear, thank you!
